_pkgdown.yml

```diff
@@ -1,15 +1,36 @@
-url: https://cynkra.github.io/fledge
+url: https://fledge.cynkra.com
 
 template:
-  params:
-    bootswatch: flatly # https://bootswatch.com/flatly/
+  package: cynkratemplate
+
+destination: docs
 
 development:
   mode: auto
 
+navbar:
+  structure:
+    left: [news, articles]
+    right: [reference, projects, consulting, github]
+  components:
+    home: ~
+    reference:
+      text: REFERENCE
+      href: reference/index.html
+    projects:
+      text: PROJECTS
+      href: https://www.cynkra.com/projects.html
+    consulting:
+      text: CONSULTING
+      href: https://www.cynkra.com/consulting.html
+
 authors:
```

Comment:
@pat-s you can actually add common authors to cynkratemplate directly see e.g. https://github.com/tidyverse/tidytemplate/blob/c584a6e1b95df94d63070c760096d838a3eed937/inst/pkgdown/BS5/_pkgdown.yml#L21